recipes/util-linux-libuuid/all/conanfile.py

```diff
@@ -23,10 +23,12 @@ class UtilLinuxLibuuidConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "with_python_bindings": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "with_python_bindings": False,
```

Comment:
```suggestion
        "with_python_bindings": True,
```

Unless there's a good reason to, let's try to avoid changing the default value for current users once they get this new revision